game_patch/rf/multi.h

```diff
@@ -88,6 +92,16 @@ namespace rf
         NG_TYPE_TEAMDM = 2,
     };
 
+    inline std::string_view multi_game_type_name(const NetGameType game_type) {
```

Comment:
why did you put those functions (`multi_game_type_name` and `multi_num_spawned_players`) into `rf` directory? This directory is supposed to only contain addresses/type definitions for RF. Please move them to `multi_scoreboard.cpp`

Reply:
My reasoning  is `rf` should be like an sdk or library e.g. if `rf` were a crate it would contain these small and simple functions which work well in this namespace. These functions would be expected in `rf` from a developer's point of view so it is intuitive and consistent. They do not properly belong to the namespace `dash`.

Reply:
Currently the criteria for putting something in `rf` is pretty simple - stuff related to internal memory structure of RF process goes there, e.g. structure definitions and addresses. Criteria like "small and simple functions" is pretty vague. Should every function below certain number of lines go there? Should functions like `bm_bytes_per_pixel` go there? Only big functions in other directories? I would rather put everything in functionality related modules, e.g. all `bm` stuff in one place. Not sure if `rf` module as it is now makes much sense, but putting there more code is something I do not like